File: lecture_notes/08 collections/4-concat.js
'use strict';

const schoolAges = [10, 12, 15, 15];
const studentAges = [17, 18, 18, 19, 20];
const NewArray = schoolAges.concat(studentAges);
const first = NewArray[0];
const last = NewArray[NewArray.length - 1];


console.log({ first, last });

File: lecture_notes/08 collections/4-concat.py
# Implementation

get_first_and_last = lambda array: (array[0], array[-1])
concat = lambda arr1, arr2: arr1 + arr2

# Usage

schoolAges = [10, 12, 15, 15]
studentAges = [17, 18, 18, 19, 20]
ages = concat(schoolAges, studentAges)
first, last = get_first_and_last(ages)

print({'first': first, 'last': last})
